promtail: add support for extra args (#259)
grafana/grafana-ansible-collection: roles/promtail/README.md

- `acl`: Root-less mode, utilizing ACL permission model to read target log files.
- `root`: Root mode, where Promtail runs as root and ACL configuration is skipped.

```yaml

> promtail_extra_args: ""

Reviewer: ```suggestion
promtail_extra_flags: []
```
These are flags, so let's use proper naming.

```

> Additional arguments/flags to be passed to each call of Promtail, such as `-config.expand-env=true`. Multiple arguments should be separated by a space.

Reviewer: ```suggestion
Additional flags to be passed to the Promtail systemd unit.
```
